test/REXUniswapV3Market.test.ts

```diff
@@ -96,100 +97,7 @@ describe('REXUniswapV3Market', () => {
-        appBalances = { ethx: [], usdcx: [], ric: [], maticx: [], rexshirt: [] };
-        ownerBalances = { ethx: [], usdcx: [], ric: [], maticx: [], rexshirt: [] };
-        aliceBalances = { ethx: [], usdcx: [], ric: [], maticx: [], rexshirt: [] };
-        bobBalances = { ethx: [], usdcx: [], ric: [], maticx: [], rexshirt: [] };
-        carlBalances = { ethx: [], usdcx: [], ric: [], maticx: [], rexshirt: [] };
-        karenBalances = { ethx: [], usdcx: [], ric: [], maticx: [], rexshirt: [] };
-    }
-
-    async function approveSubscriptions(tokensAndIDAIndexes: superTokenIDAIndex[], signers: SignerWithAddress[]) {
-        console.log("  ======== Inside approveSubscriptions ===========");
-        let tokenIndex: number;
-        for (let i = 0; i < signers.length; i++) {
-            for (let j = 0; j < tokensAndIDAIndexes.length; j++) {
-                tokenIndex = tokensAndIDAIndexes[j].IDAIndex;
-                await sf.idaV1
-                    .approveSubscription({
-                        indexId: tokenIndex.toString(),
-                        superToken: tokensAndIDAIndexes[j].token.address,
-                        publisher: market.address,
-                        userData: "0x",
-                    })
-                    .exec(signers[i]);
-                console.log("====== ", i, " subscription to token ", j, " approved =======");
-            }
-        }
-    }
-
-    async function checkBalance(user: SignerWithAddress, name: string) {
-        console.log(" checkBalance START ======== Balance of ", name, " with address: ", user.address, " ============= ");
-        let balanceEthx = await ricochetETHx.balanceOf({
-            account: user.address, providerOrSigner: provider
-        });
-        let balanceUsdcx = await ricochetUSDCx.balanceOf({
-            account: user.address, providerOrSigner: provider
-        });
-        let balanceRic = await ricochetRIC.balanceOf({
-            account: user.address, providerOrSigner: provider
-        });
-        let balanceMatic = await ricochetMATICx.balanceOf({
-            account: user.address, providerOrSigner: provider
-        });
-        console.log("Balance in ETHX: ", balanceEthx);
-        console.log("Balance in USDCX: ", balanceUsdcx);
-        console.log("Balance in RIC: ", balanceRic);
-        console.log("Balance in MATICx: ", balanceMatic);
-        console.log(" checkBalance END ====================================================== ");
-    }
-
-    async function delta(account: SignerWithAddress, balances: any) {
-        const len = balances.ethx.length;
-        return {
-            ethx: balances.ethx[len - 1] - balances.ethx[len - 2],
-            usdcx: balances.usdcx[len - 1] - balances.usdcx[len - 2],
-            ric: balances.ric[len - 1] - balances.ric[len - 2],
-            maticx: balances.maticx[len - 1] - balances.maticx[len - 2],
-            rexshirt: balances.rexshirt[len - 1] - balances.rexshirt[len - 2]
-        }
-    }
```

Comment:
You will also need to remove this code in the other files where it exists, check out the other `*.test.ts` files